internal/cmd/manager/controller/controller.go

```diff
@@ -102,9 +103,18 @@ func RunController(
 ) error {
 	ctx := context.Background()
 
+	var operatorHash string
```

Comment:
I would replace lines from 106 to 112 with

```
	operatorHash, err := executablehash.Get()
	if err != nil {
		setupLog.Error(err, "could not get executable hash")
		return err
	}
```

Reply:
👍

Reply:
I would not `return err`. IMO this is a "best effort" thing.

Reply:
If we have an error here, something really bad is happening and I would exit.
But yes, I would say return or not is debatable :)